include/models.h

```diff
@@ -70,6 +76,7 @@ class Model {
     std::vector<int32_t> inputIds;
     int batchSize;
     int seqLen;
+    int vocabSize_;
```

Comment:
Why do we need vocabSize in Model class?
if needed, could you pls make it in same naming style?

Reply:
Done